cmake/core-files.cmake

```diff
@@ -444,13 +444,16 @@ set(MBGL_CORE_FILES
     include/mbgl/util/geo.hpp
     include/mbgl/util/geojson.hpp
     include/mbgl/util/geometry.hpp
+    include/mbgl/util/ignore.hpp
     include/mbgl/util/image.hpp
+    include/mbgl/util/interpolate.hpp
     include/mbgl/util/logging.hpp
     include/mbgl/util/noncopyable.hpp
     include/mbgl/util/optional.hpp
     include/mbgl/util/platform.hpp
     include/mbgl/util/projection.hpp
     include/mbgl/util/range.hpp
+    include/mbgl/util/rapidjson.hpp
```

Comment:
It's used by Qt SDK bindings, specifically https://github.com/mapbox/mapbox-gl-native/blob/d3ca6c67f1b8fbf5d71c307e71ba89a483ec75bc/platform/qt/src/qt_geojson.hpp#L5-L5. @tmpsantos @brunoabinader do you see a way to keep it private?